### PR TITLE
Add isort+black interopability

### PR DESCRIPTION
**Pull request by awichmann-mintel**
Followed [these recommendations](https://github.com/psf/black#how-black-wraps-lines)

### CODE REVIEW

File: modules/python/Makefile.style [hunk 1]
```diff
@@ -1,5 +1,5 @@
 BLACK=$(call which,black)
-ISORT=$(call which,isort)
+ISORT=$(call which,isort) --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=88
```
Comment (charlieparkes) [inline modules/python/Makefile.style:2]: My expectation was that you should have all this in your setup.cfg or .isort.cfg

**Comment by charlieparkes** [comment]
in `setup.cfg` or `.isort.cfg`
```
[isort]
multi_line_output=3
include_trailing_comma=True
force_grid_wrap=0
use_parentheses=True
line_length=88

skip_glob = */.tox/*, */.venv/*
not_skip = __init__.py
```

**Comment by charlieparkes** [comment]
Trying to weigh whether we should stick with expecting the user to configure or force these settings through flags... 🤔

**Comment by awichmann-mintel** [comment]
> Trying to weigh whether we should stick with expecting the user to configure or force these settings through flags...

So, it's up to the user to create an isort config? Personally, I think we should have a common config and it could live in build-harness

**Comment by charlieparkes** [comment]
https://xkcd.com/1339/

Most repos already do have an .isort.cfg. The problem with assuming

https://gitlab.com/mintel/everest/cookiecutter-python-project/-/blob/master/%7B%7Bcookiecutter.project_hyphen_name%7D%7D/.isort.cfg

**Comment by charlieparkes** [comment]
Ok, 🤔 I think we can do this as you've laid it out since we're already assuming this is true by running black immediately after.

We brainstormed a few other ideas, but I think this should work for everyone rn.

**Review by charlieparkes — APPROVED** [review]

**Pull request merged**